source/features/parse-backticks.tsx

```diff
@@ -15,18 +15,28 @@ function initGeneral(): void {
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span' // Pinned Issues
+		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'blame-commit-message a' // `isBlame`
 	])) {
```

Comment:
I meant this
```suggestion
	].map(selector => selector + ':not(.rgh-parse-backticks)'))) {
		title.classList.add('rgh-parse-backticks');
```



- [ ] Merge this info initGeneral's array
- [ ] Rename initGeneral to parseAll or something
- [x] Add `.map(selector => selector + ':not(.rgh-parse-backticks)')` to that array
- [x] Actually add the class inside the loop

Reply:
>Actually add the class inside the loop

What did you mean by that? Adding that will add a class to everything! did you mean t add `rgh-parse**d**-backticks`

Reply:
[Here](https://github.com/sindresorhus/refined-github/pull/2947/commits/79e1dbbdeab966df45e3efbc5d4b5e4c8f1255d0#diff-7b0a2c8650978a85cebddd08a46e40ba) you have a selector with `a:not(.rgh-parse-commits)`

That never happens.

`rgh-parse-commits` exists exclusively on `code`, so if you want a loop like this ("select unparsed items, then parse them") you must add the "parsed" marker. _That's_ what I meant by _"Actually add the class"_

In this case I didn't notice that `rgh-parse-backticks` was already used by `code`, so I added it on the element that was being looped, as only **that** works.

---

Since `rgh-parse-backticks` is already used by `code`, yes, let's use a different class (just just parse**d**-backticks, it's too similar), but you still need to add it in this loop.

Reply:
Got you. Will try later tonight

Reply:
Should be done.

Good night